packages/react-adminer/src/components/TableFIlter.tsx

```diff
@@ -115,7 +184,29 @@ const TableFilter: React.FC<Props> = ({ setWhere, setPage, config }): any => {
 							</Col>
 						))}
 					</Row>
-					<Right>
+					<LineSpaceBetween>
+						<Button type="primary" onClick={showModal}>
```

Comment:
Pro tohle bych určitě udělal extra komponentu a řešil to v ní : `AddToFavoriteButton` - v ní bude button, v ní bude modál, v ní bude logika na přidání, atd.